Backend/Tools/rssFeedTimeConverter.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+
+
+def convert(time):
+    time = time.split(" ")
+    day = time[1]
+    month = time[2]
+    year = time[3]
+    hourMinute = time[4]
+    timezone = time[5]
```

Comment:
These variable definitions are unnecessary. To save some storage please rewrite this funtion to use the direct list or create a tuple out of the strings you need to use and document in a comment which postition corresponds to which string.